src/pages/Main.tsx

```diff
@@ -1,22 +1,28 @@
-import MainPageContent from '../components/MainPageContent';
-import BottomNavBar from '../components/BottomNavBar';
+import MainPageContent from '../components/Main/MainPageContent';
 import { SearchIcon } from '@chakra-ui/icons';
 import { Flex, Text } from '@chakra-ui/react';
 
+import { Link } from 'react-router-dom';
+import { ROUTES } from '../constants/routes';
+
 function Main() {
   return (
     <>
       <Flex h="60px" p="0 15px" align="center">
         <Text fontSize="24px" flexGrow="1">
           Campers
         </Text>
-        {/* 검색페이지로 라우트 */}
-        <SearchIcon boxSize={8} cursor="pointer" p="5px" borderRadius="15px" />
+        <Link to={ROUTES.SEARCH}>
+          <SearchIcon
+            boxSize="8"
```

Comment:
```suggestion
            boxSize={8}
```

괄호 제거를 왜 하신거죠? 다시 넣어주시길 부탁드립니다... 😂